server/lib/sources/mock.js

```diff
@@ -1,7 +1,9 @@
 // This modules mocks output generated by weboob.
 import moment from 'moment' ;
 
-import { makeLogger } from '../../helpers';
+import { makeLogger, KError } from '../../helpers';
+let errors = require('../../shared/errors.json');
```

Comment:
Well i copied what is already done for importing error codes. As there is no export in error.json i think we cannot.